wallet/res/values/strings.xml

```diff
@@ -366,10 +366,7 @@
     <string name="dash_slogan">Instant - Private - Secure</string>
     <string name="dash_wallet">Wallet</string>
 
-    <!-- TODO: Remove or change this placeholder text -->
-    <string name="hello_blank_fragment">Hello blank fragment</string>
-
-    <string name="activity_security_backup_wallet">Back Up Wallet to File</string>
+    <string name="activity_security_backup_wallet">Backup Wallet to File</string>
```

Comment:
"Back Up" is a verb and proper grammar.  "Backup" is a noun.  I changed this to "Back Up" in #289 last week.  I can fix this in my translation alignment PR #302.